config/authors/isaacadariku.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Isaac Adariku",
+  "bio": "",
```

Comment:
Isaac has spent years with interest in computer intelligence and global activities in the IT world. He is a Lead Software Engineer, Mentor and Community Organizer.